### DISCUSSION TITLE
Error: node_modules/exceljs/index.d.ts:1466:22

### DISCUSSION THREAD

**Discussion by noblemfd** [exceljs/exceljs#1841] [General]
I have been on this for some days now. I am using exceljs package

Angular -version: 12.0.5
    "@types/file-saver": "^2.0.3",
    "exceljs": "^4.3.0",
    "file-saver": "^2.0.5",

  "devDependencies": {
    "@angular-devkit/build-angular": "^12.0.5",
    "@angular/cli": "~12.0.5",
    "@angular/compiler-cli": "^12.0.5",
    "@types/jasmine": "~3.6.0",
    "@types/jquery": "^3.5.5",
    "@types/node": "^12.11.1",
    "codelyzer": "^6.0.0",
    "jasmine-core": "~3.6.0",
    "jasmine-spec-reporter": "~5.0.0",
    "karma": "~6.3.4",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage": "~2.0.3",
    "karma-jasmine": "~4.0.0",
    "karma-jasmine-html-reporter": "^1.5.0",
    "protractor": "~7.0.0",
    "ts-node": "~8.3.0",
    "tslint": "~6.1.0",
    "typescript": "^4.2.4"
  }

tsconfig.json:

    "types": ["node"],
    "importHelpers": true,
    "target": "es2015",
    "module": "es2020",
    "lib": [
      "es2018",
      "dom"
    ],
    "paths": {
      "exceljs": [
        "node_modules/exceljs/dist/exceljs.min"
      ]
    },
    "typeRoots": [
      "node_modules/@types"
    ],

I have added all the necessary things. But as soon as I :

Error: node_modules/exceljs/index.d.ts:1466:22 - error TS2307: Cannot find module 'stream' or its corresponding type declarations.

1466  read(stream: import('stream').Stream): Promise<Workbook>;
                          ~~~~~~~~


Error: node_modules/exceljs/index.d.ts:1487:23 - error TS2307: Cannot find module 'stream' or its corresponding type declarations.

1487  write(stream: import('stream').Stream, options?: Partial<XlsxWriteOptions>): Promise<void>;
                           ~~~~~~~~

Error: node_modules/exceljs/index.d.ts:1574:22 - error TS2307: Cannot find module 'stream' or its corresponding type declarations.

1574  read(stream: import('stream').Stream, options?: Partial<CsvReadOptions>): Promise<Worksheet>;
                          ~~~~~~~~


Error: node_modules/exceljs/index.d.ts:1594:23 - error TS2307: Cannot find module 'stream' or its corresponding type declarations.

1594  write(stream: import('stream').Stream, options?: Partial<CsvWriteOptions>): Promise<void>;
                           ~~~~~~~~

Error: node_modules/exceljs/index.d.ts:1898:19 - error TS2307: Cannot find module 'stream' or its corresponding type declarations.

1898    stream: import('stream').Stream;
                       ~~~~~~~~


Error: node_modules/exceljs/index.d.ts:1995:39 - error TS2307: Cannot find module 'stream' or its corresponding type declarations.

1995    constructor(input: string | import('stream').Stream, options: Partial<WorkbookStreamReaderOptions>);
                                           ~~~~~~~~

Error: node_modules/exceljs/index.d.ts:2004:18 - error TS2307: Cannot find module 'stream' or its corresponding type declarations.

2004    entry: import('stream').Stream;

I got this error:

declare const ExcelJS: any;

How do I get this sorted out?

Thanks

**Comment by skypesky** [comment]
@noblemfd 
hello, noblemfd
I have the same problem, The following links may help you.
[solution](https://github.com/exceljs/exceljs/issues/1323#issuecomment-684463556)

> tips: my @types/node version is ^12.11.1
